tools/metrics/histograms/metadata/cobalt/histograms.xml

```diff
@@ -31,6 +31,21 @@ Always run the pretty print utility on this file after editing:
 
 <histograms>
 
+<!-- The names should match media::GetCodecNameForUMA(). -->
+
+<variants name="VideoCodec">
+  <variant name="AV1"/>
```

Comment:
Per https://chromium.googlesource.com/chromium/src/tools/+/HEAD/metrics/histograms/README.md#patterned-histograms, I believe these go in the histogram definition as a `<token key...`.  Same as you did with `Pipeline`